packages/ui-kit/src/components/card-elements.tsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import * as designSystem from '../design-system';
+import { css } from '@emotion/react';
+
+/**
+ * the docs ui kit provides the basic elements to compose card UIs but does not provide
+ * the finished card, which would imply how Links are handled and would imply certain context
+ *
+ * A full composition of a card could look like:
+ * <CardContainer {...props}>
+ *  <Link>
+ *    <StackContainer>
+ *        <ImageContainer>{props.image}</ImageContainer>
+ *          <StackContainer>
+ *            <IconContainer>{props.icon}</IconContainer>}
+ *            <StackContainer scale="s">
+ *              <Title smallTitle={props.smallTitle}>{props.title}</Title>
+ *                <BodyContainer>
+ *                  <BodyContent clickable={props.clickable}>
+ *                    {props.children}
+ *                  </BodyContent>
+ *                </BodyContainer>
+ *                <ReadMoreContainer>
+ *                  <ReadMore>
+ *                    <Link href={props.href} noUnderline>
+ *                        {props.textLink}
+ *                    </Link>
+ *                  </ReadMore>
+ *                </ReadMoreContainer>
+ *            </StackContainer>
+ *        </StackContainer>
+ *      </StackContainer>
+ *    </Link>
+ *  </CardContainer>
```

Comment:
Nit: Is this the right place where we should document this?

Reply:
No, not for the long run. But we do not have any place  UI Kit documentation yet so I found it better to document it at all than not.  

I did not want to make this a full UI kit documentation project.  We would have to choose between the kit docs or building a storybook like the MC Ui kit documentation